src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbycommentfilter/Example3.java

```diff
@@ -2,7 +2,8 @@
 <module name="Checker">
   <module name="TreeWalker">
     <module name="SuppressWithNearbyCommentFilter">
-      <property name="commentFormat" value="OK to catch (\w+) here"/>
+      <property name="commentFormat"
+          value="ok,?\s*(?:allowed\s+)?to catch (\w+) here"/>
```

Comment:
```java
      <property name="commentFormat" value="ok,?\s*(?:allowed\s+)?to catch (\w+) here"/>
```
every item is non-wrapped, except of this now. Its not so long and complex we need to split, or do we?

If we keep split, might align block lvl like above:
```suggestion
      <property name="commentFormat"
                value="ok,?\s*(?:allowed\s+)?to catch (\w+) here"/>
```
<img width="843" alt="image" src="https://github.com/user-attachments/assets/e7bb94b5-ecc2-4e21-91fd-3d77e35cb0d7" />